sklearn/ensemble/_hist_gradient_boosting/tests/test_loss.py

```diff
@@ -78,7 +86,7 @@ def fprime2(x):
 
     optimum = newton(func, x0=x0, fprime=fprime, fprime2=fprime2)
     assert np.allclose(loss.inverse_link_function(optimum), y_true)
-    assert np.allclose(loss.pointwise_loss(y_true, optimum), 0)
+    assert np.allclose(func(optimum), 0)
```

Comment:
Not asking to change, but all these should have been `assert_allclose` as that produces nicer traceback on failure I think? With possible adjusted tolerances as they are not the same between `np.allclose` and `assert_allclose`.

Reply:
This kind of discussion pops up every once in a while, I've asked, there's no consensus.

https://github.com/scikit-learn/scikit-learn/issues/13180

Honestly, I don't really care anymore.